src/transformers/models/opt/modeling_opt.py

```diff
@@ -123,6 +125,9 @@ def forward(
         self,
         hidden_states: torch.Tensor,
         key_value_states: Optional[torch.Tensor] = None,
+        position_ids: Optional[
+            torch.Tensor
+        ] = None,  # isn't needed in normal attention, but needed in flash attention so to keep the signature same
```

Comment:
should be added at the end of the list of args

Reply:
OK, will change those. 

about the forward call for the embedding layer. I think it has to take position ids as an argument. otherwise it will not work with packed sentences.